api/app/Services/OpenIdBearerTokenService.php

```diff
@@ -85,7 +96,9 @@ private function getConfiguration(string $keyId) : ?Configuration
         $pem = RSAKey::createFromJWK($jwk)->toPEM();
         $config = Configuration::forAsymmetricSigner(
             $signer,
-            InMemory::empty(), // Private key is only used for generating tokens, which is not being done here, therefore empty is used.
+            InMemory::base64Encoded(
+                'hiG8DlOKvtih6AxlZn5XKImZ06yu8I3mkOzaJrEuW8yAv8Jnkw330uMt8AEqQ5LB'
```

Comment:
You lost the comment that was here.  Please add it back in, and add the reference to URL where this "junk" key came from.